<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeamCentre</class>
 <widget class="QWidget" name="BeamCentre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Beam Centre Finder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_26">
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Run</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>146</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="run_button">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="clear_log_button">
           <property name="text">
            <string>Clear Log</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>146</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Radius Limits(mm)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>from</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Max iterations</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Tolerance</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Find Beam Centre for:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Left/Right</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>Up/Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Start Position</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0" colspan="4">
          <widget class="QLineEdit" name="hab_pos1_lineEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select Row:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="select_row_combo_box"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Starting Position LAB</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QLineEdit" name="hab_pos2_lineEdit"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Starting Position HAB</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QLineEdit" name="lab_pos1_lineEdit"/>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="lab_pos2_lineEdit"/>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="select_row_push_button">
           <property name="text">
            <string>Update Rows</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
